README.md

```diff
@@ -0,0 +1,39 @@
+# WooCommerce Stripe Payment Gateway
+
+This is the official feature plugin for accepting payments via the Stripe payment gateway on a WooCommerce store.
+
+## Dependencies
+
+-   WooCommerce
+
+## Version support policy
```

Comment:
Should we capitalize "Version Support Policy" for consistency with other section titles?

Reply:
Right! Updated it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4031/commits/3985a5ead0bdbad13435fb4717ea76cc561d778d